### PR TITLE
[Validator] added reviewed Lithuanian and Dutch translations

### LINKED ISSUE #53295: Missing translations for Lithuanian (lt)
Hello,

There are some missing Lithuanian (lt) translations and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Lithuanian (lt) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf)

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by thunderer**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | Fix #53295 
| License       | MIT

Reviewed translations with help from a professional translator. No issue open for Dutch?

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf [hunk 1]
```diff
@@ -432,11 +432,11 @@
             </trans-unit>
             <trans-unit id="111">
                 <source>The detected character encoding is invalid ({{ detected }}). Allowed encodings are {{ encodings }}.</source>
-                <target state="needs-review-translation">Nustatyta simbolių koduotė yra netinkama ({{ detected }}). Leidžiamos koduotės yra {{ encodings }}.</target>
+                <target>Aptikta simbolių koduotė neteisinga ({{ detected }}). Leidžiamos koduotės {{ encodings }}.</target>
             </trans-unit>
             <trans-unit id="112">
                 <source>This value is not a valid MAC address.</source>
-                <target state="needs-review-translation">Ši vertė nėra galiojantis MAC adresas.</target>
+                <target>Tai nėra teisingas MAC adresas.</target>
```
Comment (norkunas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf]:
```suggestion
                <target>Ši reikšmė nėra teisingas MAC adresas.</target>
```

Comment (thunderer) [reply src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf]: Applied, thank you!

Comment (thunderer) [reply src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf]: @norkunas could you please approve the PR?

Comment (norkunas) [reply src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf]: Sure,but only for lt translation :)

**Review by norkunas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @thunderer.

**Pull request merged**